tests/lora/utils.py

```diff
@@ -2187,6 +2188,17 @@ def test_correct_lora_configs_with_different_ranks(self):
         self.assertTrue(not np.allclose(original_output, lora_output_diff_alpha, atol=1e-3, rtol=1e-3))
         self.assertTrue(not np.allclose(lora_output_diff_alpha, lora_output_same_rank, atol=1e-3, rtol=1e-3))
 
+    @property
+    def supports_text_encoder_lora(self):
+        return (
+            len(
+                {"text_encoder", "text_encoder_2", "text_encoder_3"}.intersection(
+                    self.pipeline_class._lora_loadable_modules
+                )
+            )
+            != 0
+        )
```

Comment:
I guess this is not being used? So, maybe we can remove?

Reply:
Done in https://github.com/huggingface/diffusers/pull/10216/commits/e97a83ea7ed53b1ed27834bcbe390bbbc6ab28b3.